### ISSUE TITLE
Missing direct file access protection in PHP file

### ISSUE THREAD

**Issue by mehrazmorshed** [equalizedigital/accessibility-new-window-warnings#25]
Missing direct file access protection in `accessibility-new-window-warnings.php` file.

PHP file should prevent direct access.
